infra-as-code/bicep/orchestration/hubPeeredSpoke/hubPeeredSpoke.bicep

```diff
@@ -17,7 +17,7 @@ param parTopLevelManagementGroupPrefix string = 'alz'
 @maxLength(10)
 param parTopLevelManagementGroupSuffix string = ''
 
-@sys.description('Subscription Id to the Virtual Network Hub object. Default: Empty String')
```

Comment:
It confused me alot that this is called Hub. It is the subscriptionId where the spoke is placed.
Or am I wrong?

Reply:
@baartch you are correct, the description should be changed as you mentioned in this PR. Good catch!